### DISCUSSION TITLE
Supporting OBS Websocket 5.x.x

### DISCUSSION THREAD

**Discussion by MatteoGheza** [josephdadams/TallyArbiter#323] [Ideas]
While working on #318 I noticed that now OBS Websocket replaced the 4.x.x protocol documentation with a new version (5.x.x) that will be released soon.
I think that TallyArbiter should be ready to support both 4.x.x and 5.x.x
The only problem is that the library that we are using supports 5.x.x in beta (and will be released as the new OBS Websocket version will be released), but doesn't support both versions.
What should we do?
@josephdadams @hrueger

**Comment by josephdadams** [comment]
I think it's fine to just include in the release notes that OBS websockets plugin 5.x.x. is required now. Otherwise it's potentially a lot of work to support both when 4.x.x is being deprecated.

**Reply by hrueger** [reply]
I agree. We could install multiple versions of the same package in two different folder, but why not just support the latest version?
